web/src/components/Cases/CasesPlot.tsx

```diff
@@ -90,16 +90,6 @@ export function CasesPlotComponent({ cluster_names, distribution }: CasesPlotPro
                     />
                   ))}
 
-                  <Area
-                    type="monotone"
-                    dataKey={CLUSTER_NAME_OTHERS}
-                    stackId="1"
-                    stroke="none"
-                    fill={theme.clusters.color.others}
-                    fillOpacity={1}
-                    isAnimationActive={false}
-                  />
-
                   <CartesianGrid stroke={theme.plot.cartesianGrid.stroke} />
```

Comment:
Here I remove the additional ad-hoc area segment from the cases plot. The same should be done for the other plot(s), when they get their own "others" data.